site/content/blog/2015-11-02-zap-newsletter-2015-november/index.md

```diff
@@ -77,28 +74,27 @@ There are 2 new options:
 
 These options take effect before other command line options such as ‘-quickurl’ so that add-ons will be installed and updated before attacking a
 target application.  
-  
 
 ###  Data Driven Content
 
 ZAP has been enhanced to add the concept of ‘data driven content’ - path elements that represent data rather than the structure of an
 application.  
 For example, these 3 URLs represent the same page but with different data:  
 
-  * https://www.example.com/app/company1/p1?ddd=eee 
-  * https://www.example.com/app/company2/p1?ddd=fff 
-  * https://www.example.com/app/company3/p1?ddd=ggg 
+  * `https://www.example.com/app/company1/p1?ddd=eee`
+  * `https://www.example.com/app/company2/p1?ddd=fff`
+  * `https://www.example.com/app/company3/p1?ddd=ggg`
 
 By default ZAP will represent them as three separate leaf nodes:  
 
-  * https://www.example.com 
-    * app 
-      * company1 
-        * GET:p1(ddd) 
-      * company2 
-        * GET:p1(ddd) 
-      * company3 
-        * GET:p1(ddd) 
+  * `https://www.example.com`
+    * `app`
+      * `company1`
+        * `GET:p1(ddd)`
+      * `company2`
+        * `GET:p1(ddd)`
+      * `company3`
+        * `GET:p1(ddd)`
```

Comment:
Would it be better to do this as a single block with indenting and no bullets? (console or text as the hint)

Reply:
There are a few other places too

Reply:
I think with bullets makes it more like the Sites tree, but happy to change.

Reply:
That's fair.